lib/DDG/Spice/BiblioSearch/GetImageForBook.pm

```diff
@@ -0,0 +1,18 @@
+package DDG::Spice::BiblioSearch::GetImageForBook;
+
+use strict;
+use DDG::Spice;
+use URI::Escape;
+
+triggers any => '///***do not trigger***///';
+
+spice to => 'https://duckduckgo.com/m.js?q=$1';
```

Comment:
You don't need a Perl module for this -- you should be able to call this endpoint directly from the JS